modules/services/sketchybar/default.nix

```diff
@@ -43,17 +54,39 @@ in
         and [example](https://github.com/FelixKratz/SketchyBar/blob/master/sketchybarrc).
       '';
     };
+
+    errorLogFile = mkOption {
+      type = types.nullOr types.path;
+      default = "/var/tmp/sketchybar.error.log";
+      example = "/Users/khaneliman/Library/Logs/sketchybar.log";
+      description = "Absolute path to log all stderr output.";
+    };
+
+    outLogFile = mkOption {
+      type = types.nullOr types.path;
+      default = "/var/tmp/sketchybar.out.log";
+      example = "/Users/khaneliman/Library/Logs/sketchybar.log";
+      description = "Absolute path to log all stdout output.";
+    };
```

Comment:
Ideally this would match upstream as well, which I believe is under `/var/log`:

https://github.com/FelixKratz/homebrew-formulae/blob/8cd8fae90b2e154e3fa3092cc5f8e3a6e963acd0/sketchybar.rb#L61-L68

but this is a user agent and that's not a user-owned directory. I grepped and we don't seem to have a consistent location. Personally I would use `/tmp`, but this is fine.